pages/linux/kernel-install.md

```diff
@@ -0,0 +1,16 @@
+# kernel-install
+
+> Add and remove kernel and initrd images to and from /boot.
+> More information: <https://man.archlinux.org/man/kernel-install.8>.
```

Comment:
```suggestion
> More information: <https://manned.org/kernel-install.8>.
```
We use Manned as the default fallback when there isn't an author's documentation. (And this command isn't an Arch Linux project so we can use manned)

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links

Reply:
Change added.